mantidproject/mantidimaging
# New filter window

Closes #138

Replaces the filter dialog with a new window that follows the MVP design pattern and contains the filter application logic (which was previously contained in the stack visualiser presenter).

To test:
- Load `demoimages/sample`
- Apply the background correction with the flat and dark images in `demoimages` (tests pre and post processing operations)
- Select a ROI containing the sample and apply the Crop Coordinates filter (tests auto parameters from stack)
- Try invalid input, missing data, etc. Stack traces should appear in the log but the UI should not crash.
